static/katex.less

```diff
@@ -232,8 +232,25 @@
         & + .mop.mtight { margin-left: @thinspace; }
     }
 
+    .vlist-t {
+        display: inline-table;
+        table-layout: fixed;
```

Comment:
Reading https://developer.mozilla.org/en-US/docs/Web/CSS/table-layout it says that `table-layout: fixed` bases the width of the table on the first row.  It looks like the first row is always contains the content for the vlist.  Is that right?